core/Plugin/Manager.php

```diff
@@ -1191,6 +1191,8 @@ private function executePluginDeactivate($pluginName)
      */
     private function unloadPluginFromMemory($pluginName)
     {
+        $this->unloadPlugin($pluginName);
+
```

Comment:
Just realize it was re `unloadPluginFromMemory`... well, I can have a look if there's maybe a better name that makes it clearer. The name was already there...